### PR TITLE
Features/input/number

### PR DESCRIPTION
**Pull request by yinchyy**

### CODE REVIEW

File: src/Button/Button.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export type ButtonProps = {
   /**
    * Should button have regular look, be outlined or serve as a link?
    */
-  style?: 'standard' | 'outlined' | 'link';
+  buttonStyle?: 'standard' | 'outlined' | 'link';
```
Comment (pilotpirxie) [inline src/Button/Button.tsx:26]: would be great to have one naming convention over as many components as possible. I'm not saying this ^ is wrong, just to consideration

File: src/Number/Number.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <div
+      className={cx(styles.textInput, styles.fontRegular, {
+        [styles.fullWidth]: fullWidth,
+      })}
+    >
+      <input
+        type="number"
+        id={id}
+        className={cx(styles.styleNone, styles.fontRegular)}
+        onFocus={(e) => { e.currentTarget.placeholder = focusedPlaceholder; }}
```
Comment (pilotpirxie) [inline src/Number/Number.tsx:51]: tbh different placeholder on focus is a nice feature but should not be required. And if it's not provided it should use the default placeholder

File: src/Number/Number.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ * Number component
+ */
+export function Number({
+  id,
+  fullWidth = false,
+  focusedPlaceholder = 'Type something...',
+  disabled = false,
+  ...props
+}: NumberProps) {
+  const inputRef = useRef<any>(null);
```
Comment (pilotpirxie) [inline src/Number/Number.tsx:39]: don't use any, provide strong type

File: src/Button/Button.tsx (hunk 1 above)
Comment (yinchyy) [reply src/Button/Button.tsx]: resolved [ca7e8bb](https://github.com/pilotpirxie/react-toucan-ui/pull/11/commits/ca7e8bbf1789c118a1e571464b02f65281ec0352)

File: src/Number/Number.tsx (hunk 2 above)
Comment (yinchyy) [reply src/Number/Number.tsx:51]: resolved [ca7e8bb](https://github.com/pilotpirxie/react-toucan-ui/pull/11/commits/ca7e8bbf1789c118a1e571464b02f65281ec0352)

File: src/Number/Number.tsx (hunk 3 above)
Comment (yinchyy) [reply src/Number/Number.tsx]: resolved [fb25993](https://github.com/pilotpirxie/react-toucan-ui/pull/11/commits/fb25993a72ddbe90b08d35a0f50e27d1298107b3)

File: src/Text/Text.tsx [hunk 4]
```diff
@@ -27,10 +31,11 @@ export type TextProps = {
 export function Text({
   id,
   fullWidth = false,
-  focusedPlaceholder = 'Type something...',
+  disabled = false,
   ...props
 }: TextProps) {
   const placeholder = typeof props.placeholder !== 'undefined' ? props.placeholder : 'Text';
+  const focusedPlaceholder = typeof props.focusedPlaceholder !== 'undefined' ? props.focusedPlaceholder : placeholder;
```
Comment (pilotpirxie) [inline src/Text/Text.tsx:38]: hmm
wouldn't 
```
const focusedPlaceholder = props.focusedPlaceholder || placeholder;
``` 
be sufficient? 
```
undefined || 'test' // 'test'
```

File: src/Number/Number.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+ */
+export function Number({
+  id,
+  fullWidth = false,
+  disabled = false,
+  ...props
+}: NumberProps) {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const placeholder = typeof props.placeholder !== 'undefined' ? props.placeholder : 'Number';
+  const focusedPlaceholder = typeof props.focusedPlaceholder !== 'undefined' ? props.focusedPlaceholder : placeholder;
```
Comment (pilotpirxie) [inline src/Number/Number.tsx:40]: as above

File: src/Button/Button.tsx [hunk 6]
```diff
@@ -40,7 +40,7 @@ export type ButtonProps = {
 export function Button({
   size = 'medium',
   color = 'primary',
-  style = 'standard',
+  styling = 'standard',
```
Comment (pilotpirxie) [inline src/Button/Button.tsx:43]: :+1:

**Review by pilotpirxie — APPROVED** [review]

File: src/Text/Text.tsx (hunk 4 above)
Comment (yinchyy) [reply src/Text/Text.tsx]: resolved [649e925](https://github.com/pilotpirxie/react-toucan-ui/pull/11/commits/649e92513d8c3f9c23a7290aec03d5742586baff)

File: src/Number/Number.tsx (hunk 5 above)
Comment (yinchyy) [reply src/Number/Number.tsx]: resolved [649e925](https://github.com/pilotpirxie/react-toucan-ui/pull/11/commits/649e92513d8c3f9c23a7290aec03d5742586baff)

**Pull request merged**